tests/00-kubectl-get-namespace/get-ns-test

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+namespace="$1"
+github_team="$2"
+
+if kubectl auth can-i get namespace --namespace "${namespace}" --as=test --as-group=github:"${github_team}" | grep  -q "yes"; then
```

Comment:
Is `as=test` doing this as user "test"? How do you get that user?

Reply:
In RBAC, users and groups are not actual resources (unlike ServiceAccounts) since they're supposed to be managed by some external identity provider (through OIDC). So the values you use here can be anything. The `RoleBinding` then is what grants permissions to the users/groups. Essentially, you can use any arbitrary string.